### DISCUSSION TITLE
How to set aliases in config nu?

### DISCUSSION THREAD

**Discussion by MadSpindel** [nushell/nushell#9586] [Q&A] [answered]
I want to set persistant aliases in `config nu` but the docs are not clear on how to do so.

> To make your aliases persistent they must be added to your config.nu file by running config nu to open an editor and inserting them, and then restarting nushell.

But how should I add them in the config.nu file?
https://www.nushell.sh/book/aliases.html

**Comment by amtoine** [comment, marked as answer]
@MadSpindel 
this has been asked quite a few time already, so i've added a complete example to the website :yum: 

:point_right: https://github.com/nushell/nushell.github.io/pull/973
